weaviate/collections/backups.py

```diff
@@ -0,0 +1,109 @@
+from weaviate.backup.backup import (
+    _Backup,
+    _BackupStatusReturn,
+    BackupStorage,
+)
+from weaviate.connect.connection import Connection
+
+
+class _CollectionBackup:
+    """Backup functionatility for this collection."""
+
+    def __init__(self, connection: Connection, name: str):
+        self._connection = connection
+        self._name = name
+        self._backup = _Backup(connection)
+
+    def create(
+        self, backup_id: str, backend: BackupStorage, wait_for_completion: bool = False
```

Comment:
Can the `backup_id` be auto-generated by us so that the user doesn't have to provide it or is there a risk of it colliding with an ID that is already registered in the cloud blob storage backend that is beyond the control of the client? (I.e., the client would have to do a look-up to check for no collision)

Reply:
I kept it as close as possible to the existing implementation and it was done like this before.

I think you need to know the ID to use it again afterwards - so either we force the user to provide it or we also return it to the user after creation. If we randomly generate something there is always the chance of a collision

Reply:
Looking at the implementation it looks like `backupId` is required for the modules to work so it's not something we can help on the client-side. Also I guess not many use this anyway so 🤷